test/Utilities/StyleMocking.js

```diff
@@ -0,0 +1,40 @@
+import assert from 'assert'
+
+// Domino doesn't seem to slurp up css class properties if we set them in our fixture html.
```

Comment:
This was a profound discussion with a lot of subtelties uncovered. The minutes I took for our readme (both our responsibilities) are:

- Always prefer to modify the class list instead of inline style attributes and to test accordingly
- Always prefer to check the class list and inline styles instead of computed styles
- Add integration tests for parsoid output -- thehse are indepedent of styles
- Add visual regression tests for verifying cascading styles look good

^^^ FOr these rules above, we need some nice exampels for these above. And, just in general, we some nice examples. ^^^